jenkins/pipelines/dev_e2e/android/topology_single_device.json

```diff
@@ -2,7 +2,7 @@
     "$schema": "../../../../environment/aws/topology_setup/topology_schema.json",
     "test_servers": [
         {
-            "platform": "c_android",
```

Comment:
The renaming algorithm on GitHub got this one wrong, These are two different files.